opensearch/src/main/java/org/opensearch/sql/opensearch/response/agg/FilterParser.java

```diff
@@ -1,16 +1,9 @@
 /*
- *   Licensed under the Apache License, Version 2.0 (the "License").
- *   You may not use this file except in compliance with the License.
- *   A copy of the License is located at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- *   or in the "license" file accompanying this file. This file is distributed
- *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- *   express or implied. See the License for the specific language governing
- *   permissions and limitations under the License.
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Looks like this PR has quite a few spots with the same header that Yury has mentioned [here](https://github.com/Bit-Quill/opensearch-project-sql/pull/306#discussion_r1271000393) and should probably should be looked into as well